packages/cli/src/modules/insights/entities/insights-shared.ts

```diff
@@ -11,7 +11,9 @@ export const PeriodUnitToNumber = {
 	day: 1,
 	week: 2,
 } as const;
+// TODO: rename to singular
 export type PeriodUnits = keyof typeof PeriodUnitToNumber;
+// TODO: rename to singular
 export type PeriodUnitNumbers = (typeof PeriodUnitToNumber)[PeriodUnits];
```

Comment:
Do we need to rename these now (as the comments say)?

Reply:
I renamed